### PR TITLE
Switch Windows builds to Travis

### PR DESCRIPTION
**Pull request by morris25**
This change simplifies our testing pipeline by removing Appveyor and having all tests run on TravisCI.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,10 +19,26 @@ after_success:
   - julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Codecov.submit(Codecov.process_folder())'
 jobs:
   include:
+    - os: windows
```
Comment (mattBrzezinski) [inline .travis.yml]: Can this not be simplified?

**Comment by morris25** [comment]
Note that Appveyor will continue to cause tests on this PR to fail until it is turned off.

File: .travis.yml [hunk 2]
```diff
@@ -3,11 +3,16 @@ language: julia
 os:
   - linux
   - osx
+  - windows
 julia:
   - 1.0
```
Comment (mattBrzezinski) [inline .travis.yml]: @omus thoughts on testing all these versions? Or do you think LTS and most recent release would suffice?

Comment (omus) [reply .travis.yml]: I think LTS, most recent release, and nightly is fine for most projects

**Pull request merged**